system/validator_revard/Reward-calculation.md

```diff
@@ -0,0 +1,161 @@
+
+### CAUTION
+If a validator is oversubscribed in an era, staking rewards are distributed only to the the top 512 nominators and the rest of the nominators do not receive any rewards. This is not the case for slashing! Every active nominator of the validator committing slashable offence will be slashed.
+
+# Nominators and Validator Payments
+Nominated stake allows you to "vote" for validators and share in the rewards (and slashing) without running a validator node yourself. Validators can choose to keep a percentage of the rewards due to their validator to "reimburse" themselves for the cost of running a validator node. Other than that, all rewards are shared based on the stake behind each validator. This includes the stake of the validator itself, plus any stake bonded by nominators.
+
+### INFO
+Validators set their preference as a percentage of the block reward, not an absolute number of joys. joystream's block reward is based on the total amount at stake, with the reward peaking when the amount staked is at 50% of the total supply. The commission is set as the amount taken by the validator; that is, 0% commission means that the validator does not receive any proportion of the rewards besides that owed to it from self-stake, and 100% commission means that the validator operator gets all rewards and gives none to its nominators.
+
+In the following examples, we can see the results of several different validator payment schemes and split between nominator and validator stake. We will assume a single nominator for each validator. However, there can be numerous nominators for each validator. Rewards are still distributed proportionally - for example, if the total rewards to be given to nominators is 2 joys, and there are four nominators with equal stake bonded, each will receive 0.5 joys. Note also that a single nominator may stake different validators.
+
+Each validator in the example has selected a different validator payment (that is, a percentage of the reward set aside directly for the validator before sharing with all bonded stake). The validator's payment percentage (in joys) is listed in brackets ([]) next to each validator. Note that since the validator payment is public knowledge, having a low or non-existent validator payment may attract more stake from nominators, since they know they will receive a larger reward.
+
+**Validator Set Size (v): 4**  
```

Comment:
This whole section is very hard to follow in my eyes. Also take an example of [this forum entry](https://pioneerapp.xyz/#/forum/thread/128?post=565) how a good guide can look like.
Thinking about it maybe the [validator forum category](https://pioneerapp.xyz/#/forum/category/50) may indeed be a better place for this, wdyt @bedeho?